package.json

```diff
@@ -113,6 +118,7 @@
         "merkletreejs": "^0.2.32",
         "ms": "^2.1.3",
         "multiformats": "^9.4.7",
+        "mysql": "^2.18.1",
```

Comment:
Update dkg.js version, to pull latest one from npm
move ganache-core to dev-dependencies
remove mysql dependency since we are using mysql2
Remove setTimeout we are not using it anywhere